docs/userguide/modules/getting-started/partials/deploy-db-postgresql.adoc

```diff
@@ -0,0 +1,75 @@
+== Create a PostgreSQL database instance
+
+The application is going to use a PostgreSQL database instance. 
+
+To create a database instance, you need to create the following resources:
+
++++ <details><summary> +++
+`Deployment` to run the actual database instance
++++ </summary><div> +++
+[source,yaml]
+----
+include::attachment$postgresql-deployment.yaml[tag=db-deployment]
```

Comment:
That is likely because I've shared the rendered `out/site` from the PR branch that contains the files that are linked. Those YAMLs would be actually be linked from the github pages properly, when the docs are published.